broker/setup_broker/setup_gcp.py

```diff
@@ -93,7 +93,7 @@
 PROJECT_ID = os.getenv('GOOGLE_CLOUD_PROJECT')
 
 
-def _resources(service, survey='ztf', testid='test'):
+def _resources(service, survey='ztf', testid='test', versiontag="v3_3"):
```

Comment:
Is there a easier place to define this "v3_3" string.  When this has to be updated, it will be difficult to find on line 96 of a file.

Reply:
I don't really understand what you're asking. In practice, this gets defined by the user and passed into the script. Are you wanting to more easily update the default value? I think that's a problem throughout this script... I agree that the way this entire script is setup is bad, but I don't see a reason to make a special (i.e., different than `survey` or `testid`) way to set the `versiontag` default. (I'm guessing that this script will be entirely revamped or replaced in the near future anyway, along with the rest of the setup scripts.)

Reply:
For this PR, don't change anything. 

But for design consideration in the future:

For things like this I might consider having some globals (or at least module-level variables) for defaults.  I tend to use all caps for these to indicate that they are special values.

VERSIONTAG = v3_3

They can live up at the top of the file, ideally only in one file.  It's a bit annoying to share between shell scripts and Python code, but you could at least reduce to two files.

One reason I call out versiontag in particular is that survey, testid, and versiontag each draw from a different part of the understanding.  Survey makes immediate sense, "Oh, right I need to specify the survey to listen to/load."  Testid makes a little sense if you think you're testing.  But a user has no particularly obvious way to reason about what versiontag should be.

One might also decide to not provide defaults and force to user to immediately have to make choices.